build.gradle

```diff
@@ -17,6 +17,7 @@ plugins {
     id "org.jlleitschuh.gradle.ktlint" version "9.4.0"
     id "org.jlleitschuh.gradle.ktlint-idea" version "9.4.0"
     id "io.gitlab.arturbosch.detekt" version "1.13.1"
+    id "de.marcphilipp.nexus-publish" version "0.4.0"
```

Comment:
`de.marcphilipp` does not look like an official plugin, but if it helps I'm ok with it.